app/views/shared/_top_artists.html.slim

```diff
@@ -0,0 +1,19 @@
+- if @featured_artists.empty?
+- else
+  .line-wrapper
+    .container.d-flex.justify-content-between
+      h4.title. OUR ARTISTS
+      hr.lines.w-75
+  .container.mb-5
+    .row.artists-cards
```

Comment:
These cards need to link to the artist pages

Reply:
done

Reply:
<3